tests/Endpoints/LearningMaterialTest.php

```diff
@@ -344,7 +343,7 @@ public function testPostLearningMaterialFile()
         $data['filename'] = $responseData['filename'];
 
         $postData = $data;
-        $data['mimetype'] = 'text/x-php';
+        $data['mimetype'] = 'text/plain';
```

Comment:
Was this on purpose? I wasn't able to figure out why.

Reply:
my mistake, i was messing around with something semi-related there but gave up halfway through.

```suggestion
        $data['mimetype'] = 'text/x-php';
```